gui/wxpython/startup/utils.py

```diff
@@ -55,6 +58,50 @@ def get_possible_database_path():
     return path
 
 
+def create_database_directory():
+    """Creates the standard GRASS GIS directory.
+
+    Creates database directory named grassdata in the standard location 
+    according to the platform.  
+    
+    Returns the new path as a string or None if nothing was found or created.
+    """
+    home = os.path.expanduser('~')   
+    candidates = []       
+   
+    # Candidate for case independent "grassdata" for Linux and macOS
+    if sys.platform.startswith('linux') or sys.platform.startswith('darwin'):
+        candidates.append(os.path.join(home, "grassdata"))   
+    # Candidates for case independent "grassdata" in other dirs (Windows)
+    else:  
+        candidates.append(os.path.join(home, "Documents", "grassdata"))
+        
+    # Create "grassdata" directory           
+    for candidate in candidates:                
+        try:
+            os.mkdir(candidate)
+            return candidate
+        except OSError:
+            pass
+    
+    # Temporary "grassdata" directory     
+    tmp = os.path.join(
+        tempfile.gettempdir(),
+        "grassdata_{}".format(getpass.getuser())
+    ) 
+    
+    # Check if exists, if does not, create it
+    if os.path.exists(tmp):
+        return tmp
+    else:
+        try:
+            os.mkdir(tmp)
+            return tmp
+        except OSError:
+            pass
```

Comment:
Since the true branch of the if statement has return, you don't need else here and can move the try-except block one level left. This is does not change anything, but is makes the code simpler and the same logic is already used above: The good case returns, so the other case is just whatever follows after that.